src/app/core/market/proposals-notifier/proposals-notifications.service.ts

```diff
@@ -14,6 +14,7 @@ export class ProposalsNotificationsService implements OnDestroy {
 
   log: any = Log.create('order-status-notifier.service id:' + Math.floor((Math.random() * 1000) + 1));
   public proposals: Proposal[] = [];
```

Comment:
Do we need to store proposals? Or should this not rather be a numerical value indicating the number of proposals that are new?

There doesn't seem to be much of a point in saving all of the proposals.